src/Test/Utilities/Portable/CommonTestBase.cs

```diff
@@ -608,7 +608,27 @@ internal static void VerifyClone(SemanticModel model)
                     continue;
                 }
 
-                var clonedOperation = model.CloneOperation(operation);
+                IOperation clonedOperation;
+                if (operation.Parent == null)
+                {
+                    clonedOperation = model.CloneOperation(operation);
+                }
+                else
+                {
+                    var count = 0;
+                    foreach (var child in operation.Parent.Children)
+                    {
+                        if (child == operation)
+                        {
+                            break;
+                        }
+
+                        count++;
+                    }
+
+                    // need to clone from parent since OperationTreeVerifier puts parent information in the output
+                    clonedOperation = model.CloneOperation(operation.Parent).Children.Skip(count).First();
```

Comment:
:bulb: `ElementAt(count)`

Or even better, `First(child => child == operation)` (which eliminates the `count` variable and the entire `foreach` block).

Reply:
2 are 2 different tree, can't use First(...). sure, I can use ElementAt